internal/config/config.go

```diff
@@ -111,5 +115,21 @@ func New() *Config {
 		// os.Getenv calls in other functions with these values.
 		KokoroHostRootDir: os.Getenv("KOKORO_HOST_ROOT_DIR"),
 		KokoroRootDir:     os.Getenv("KOKORO_ROOT_DIR"),
+		GitHubToken:       os.Getenv("LIBRARIAN_GITHUB_TOKEN"),
+	}
+}
+
+// IsValid ensures the values contained in a Config are valid.
+func (c *Config) IsValid() (bool, error) {
+	if err := c.validatePush(); err != nil {
+		return false, err
+	}
+	return true, nil
+}
+
+func (c *Config) validatePush() error {
+	if c.Push && c.GitHubToken == "" {
```

Comment:
I would move this inside `IsValid` since it's just one line to avoid a level of indirection (unless there's a reason for separating that I might not be seeing)

Reply:
I imagine `IsValid()` will grow to include many other validations, so separating them by "theme" would make sense. For now though, I agree that the additional method only serves as extra indirection, removed.